### PR TITLE
Elias/make slides sortable

### PR DESCRIPTION
**Pull request by elias4711**
Make slides sortable by adding a field to the cms and sort array of slides by position value

### CODE REVIEW

File: components/Carousel.js [hunk 1]
```diff
@@ -25,21 +26,24 @@ export default function Carousel({ slides }) {
         <CarouselProvider
           className={`${styles.carousel}`}
           infinite
-          isPlaying
+          loop
```
Comment (friedjoff) [inline components/Carousel.js:29]: I don't find `loop` in the documentation for `CarouselProvider`. Is this making any difference?

File: components/Carousel.js [hunk 2]
```diff
@@ -25,21 +26,24 @@ export default function Carousel({ slides }) {
...
+          playDirection="forward"
```
Comment (friedjoff) [inline components/Carousel.js:30]: Not really needed because that's the default value.
```suggestion
```

File: components/Carousel.js [hunk 3]
```diff
@@ -25,21 +26,24 @@ export default function Carousel({ slides }) {
...
           naturalSlideWidth={1600}
           naturalSlideHeight={1200}
           totalSlides={slides.length}
+          isPlaying
```
Comment (friedjoff) [inline components/Carousel.js:34]: For better readability I would keep the props in alphabetical order.

File: components/Carousel.js [hunk 4]
```diff
@@ -16,6 +16,7 @@ const CarouselKeyboardNavigation = dynamic(
 );
 
 export default function Carousel({ slides }) {
+  slides.sort((a, b) => a.position - b.position);
```
Comment (friedjoff) [inline components/Carousel.js:19]: We should sort the slides in `/pages/index.js` (see `getStaticProps`) so we don't do it on each render.

File: public/config.yml [hunk 5]
```diff
@@ -408,6 +408,11 @@ collections:
         name: "video"
         widget: "string"
         required: false
+      - label: "Position"
```
Comment (friedjoff) [inline public/config.yml:411]: For better consistency we should name this field `weight` like in other cases.

**Review by friedjoff — CHANGES_REQUESTED** [review]
Nice idea! Just a few changes to improve consistency.

**Comment by elias4711** [comment]
@friedjoff  I worked in your Points in the PR

**Review by friedjoff — APPROVED** [review]

**Pull request merged**